rpcs3/Emu/System.h

```diff
@@ -119,6 +119,8 @@ class Emulator final
 	atomic_t<u64> m_pause_amend_time{0}; // increased when resumed
 	atomic_t<u64> m_stop_ctr{0}; // Increments when emulation is stopped
 
+	u64 m_games_yml_write_ctr = 0;
```

Comment:
you can just use a bool and set it to false at the beginning of AddGamesFromDir

Reply:
A counter is independent of its users, if you want to use it to check invalidations in more than place at once you can.
With bool you are limited to a single use instance.

Reply:
You don't use it in that way though, which only overcomplicates things